src/test-support/FakeKuma.ts

```diff
@@ -61,6 +61,25 @@ export class KumaModule {
     return `${this.faker.hacker.noun()}-${deploymentId}-${podId}.${namespace}`
   }
 
+  connection<T>(_: T, i: number, arr: T[]) {
+    const connected = this.faker.date.past()
+    const times: {
+      connectTime: string,
+      disconnectTime?: string
+    } = {
+      connectTime: `${connected.toISOString().slice(0, -1)}${this.faker.number.int({ min: 1000, max: 9999 })}Z`,
+      // connectTime: '2021-07-13T08:41:04.556796688Z',
```

Comment:
**Suggestion**: Development remnant
```suggestion
```

Reply:
I left these in on purpose so you could see the format and understand what all the fake number padding was about.